src/Shapes/Shape.java

```diff
@@ -1,59 +1,74 @@
 package Shapes;
 
-import java.awt.Graphics;
-import java.awt.Point;
-import java.awt.Font;
+import java.awt.*;
```

Comment:
Please keep the below

```java
import java.awt.Graphics;
import java.awt.Point;
import java.awt.Font;
```

Reply:
If you specify only 3 packages, then the polygon class is missing, after adding it, all packages in the import are automatically substituted, how do we do it?
![1](https://github.com/1chooo/uml-editor/assets/103263545/e1765010-4ed9-4354-9acb-ffa18e28194d)
![2](https://github.com/1chooo/uml-editor/assets/103263545/d3b199c7-8d29-4004-b28a-cb6586c4d90c)
![3](https://github.com/1chooo/uml-editor/assets/103263545/06a03ccc-6f8f-4ab5-bbb6-6d34adf42b1c)
![4](https://github.com/1chooo/uml-editor/assets/103263545/55d2f13d-f9cd-45c2-b5ea-54f755b8ae2a)

Reply:
you can use

```java
import java.awt.Polygon;
```

instead of 
```java
import java.awt.*;
```

Reply:
https://github.com/1chooo/uml-editor/pull/27
new